spec/features/support/allocations/allocation_uplift_edit_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Edit an Allocation Uplift" do
+  let(:user) { create(:user, :admin) }
+  let(:provider) { create(:provider) }
+  let(:allocation) { create(:allocation, :with_allocation_uplift, provider: provider, number_of_places: 5) }
+  let(:new_allocation_uplift) { 500 }
```

Comment:
I think with feature specs we can skip the use of `let` blocks — they're a layer of indirection that don't necessarily make the spec easier to follow. The Apply [testing styleguide](https://github.com/DFE-Digital/apply-for-teacher-training/blob/0efbef5707db151b128015c710011b8685b3485b/docs/testing-styleguide.md) outlines what I think we should be aiming for.